LBHackney-IT/lbh-social-care
# unified case notes recording forms

**What**  
- unified case notes recording forms

Before:
  `/form/case-notes-recording/case-notes-recording`
  `/form/cfs-case-notes-recording/cfs-case-notes-recording`
To:
  `/people/:peopleId/case-notes-recording`

- removed `FormCaseWrapper` component, used HOC `PersonView` instead

**How to test it**
- check an **adult** `/people/234/case-notes-recording`
- check a **child** `/people/321569/case-notes-recording`
- they should show different forms
